scripts/update/jellyfin.sh

```diff
@@ -1,87 +1,87 @@
-        sed -r 's#<RequireHttps>false</RequireHttps>#<RequireHttps>true</RequireHttps>#g' -i "/etc/jellyfin/system.xml"
-        #
-        if [[ -f /etc/nginx/apps/jellyfin.conf ]]; then
-            sed -r 's#proxy_pass https://127.0.0.1:(.*);#proxy_pass https://127.0.0.1:8920;#g' -i "/etc/nginx/apps/jellyfin.conf"
-            systemctl -q restart "nginx.service"
-        fi
-        #
-        [[ -d "/home/${username}/.config/Jellyfin" ]] && rm -rf "/home/${username}/.config/Jellyfin" | :
-        [[ -d "/home/${username}/.cache/jellyfin" ]] && rm -rf "/home/${username}/.cache/jellyfin" | :
-        [[ -d "/home/${username}/.aspnet" ]] && rm -rf "/home/${username}/.aspnet" | :
-    fi
-    #
-    ## legacy end
-    #
-    # Add the jellyfin official repository and key to our installation so we can use apt-get to install it jellyfin and jellyfin-ffmepg.
-    wget -q -O - https://repo.jellyfin.org/debian/jellyfin_team.gpg.key | sudo apt-key add -
-    echo "deb [arch=$( dpkg --print-architecture )] https://repo.jellyfin.org/$(. /etc/os-release; echo $ID) $(lsb_release -cs) main" > /etc/apt/sources.list.d/jellyfin.list
-    #
-    # install jellyfin and jellyfin-ffmepg using apt functions. 
-    apt_update #forces apt refresh
-    apt_install jellyfin jellyfin-ffmpeg
-    #
-    # Configure the new jellyfin service.
-    systemctl -q stop "jellyfin.service"
-    #
-    # Add the jellyfin user to the master user's group to use our ssl certs.
-    usermod -a -G ${username} jellyfin
-    #
-    # Set the correct and required permissions of any directories we created or modified.
-    [[ -d "/home/${username}/.ssl" ]] && chown "${username}.${username}" -R "/home/${username}/.ssl" || :
-    [[ -d "/home/${username}/.ssl" ]] && chmod -R g+r "/home/${username}/.ssl" || :
-    #
-    # Set the default permissions after we have migrated our data
-    [[ -d "/etc/jellyfin" ]] && chown -R jellyfin:jellyfin "/etc/jellyfin" || :
-    [[ -f "/etc/jellyfin/logging.json" ]] && chown jellyfin:root "/etc/jellyfin/logging.json" || :
-    [[ -d "/etc/jellyfin" ]] && chown jellyfin:adm "/etc/jellyfin" || :
-    [[ -d "/var/lib/jellyfin" ]] && chown -R jellyfin:jellyfin "/var/lib/jellyfin" || :
-    [[ -d "/var/lib/jellyfin" ]] && chown jellyfin:adm "/var/lib/jellyfin" || :
-    #
-    # Reload systemd and start the service.
-    systemctl -q daemon-reload
-    systemctl -q start "jellyfin.service"
-    #
-    echo -e "\nJellyfin upgrade completed and service restarted\n"
-    #
-    exit
-
-fi
+#!/usr/bin/env bash
+#
+if [[ -f /install/.jellyin.lock ]]; then
+    #
+    # source the functions we need for this script.
+    . /etc/swizzin/sources/functions/utils
+    #
+    # Get our main user credentials using a util function.
+    username="$(_get_master_username)"
+    #
+    # disable the service to make usre the mosrt up to date and correct service file is installed via apt.
+    systemctl -q stop "jellyfin.service"
+    #
+    ## legacy start
```

Comment:
You might want to wrap the entire thing here in one condition, because this script will be executed every single time `box update` will be executed. So if anyone has jellyfin lockfile present, the entire thing will get executed

Reply:
done